extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
@jsutterfield: I still think it's better to understand the code before applying the fix. That's the only way we can pay off the technical debt and avoid such issues in the future :)

`AjaxResponse` class is used to generate the [ArticleComments response](https://github.com/Wikia/app/blob/808a769df6cf8524aa6defcab4f971367e3e3fd8/extensions/wikia/ArticleComments/ArticleComments_setup.php#L136). Try setting the cache TTL to zero (as in [WikiFactory code](https://github.com/Wikia/app/blob/808a769df6cf8524aa6defcab4f971367e3e3fd8/extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php#L120)):

```php
$response = new AjaxResponse($payload);
$response->setCacheDuration(0);
```